pkg/security/security.go

```diff
@@ -163,6 +163,10 @@ type Options struct {
 	// WorkloadRSAKeySize is the size of a private key for a workload certificate.
 	WorkloadRSAKeySize int
 
+	// Whether to enable dual-use mode, which generates a workload certificate with a CommonName identical to
```

Comment:
Same - what is added to cert should be a CA policy, not something the pods decide.